cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst

```diff
@@ -63,15 +67,24 @@ API. To do that, send an HTTP GET request::
 
    * - Parameter
      - Description
-   * - ``<namespace>``
-     - Namespace ID
-   * - ``<component-id>``
-     - ``appfabric``
    * - ``<service-id>``
-     - [to be completed]
+     - One of ``appfabric``, ``dataset.executor``, ``explore.service``, ``metrics``, ``streams``
```

Comment:
@awholegunch, @gokulavasan just confirmed that we should add ``transaction`` and ``metrics.processor`` to this list.